apps/condo/domains/notification/access/SyncDeviceService.js

```diff
@@ -8,9 +8,7 @@
  * that are managed within service itself, in order to not duplicate operation type detection heuristics logic
  * @returns {Promise<boolean>}
  */
-async function canSyncDevice ({ args: { data } }) {
-    if (!data.deviceId) return false
-
+async function canSyncDevice () {
     return true
```

Comment:
If user is authorised, but somehow deleted? I guess we should do something like:
`if (user && user.deletedAt) return false`

Reply:
fixed